README.md

```diff
@@ -1,9 +1,11 @@
 # Growing Neural Cellular Automata - Task Experiments  
 
-Based on the original work [Growing Neural Cellular Automata](https://distill.pub/2020/growing-ca/) by Alexander Mordvintsev et al.  
+Based on the original work [Growing Neural Cellular Automata](https://distill.pub/2020/growing-ca/) by Alexander Mordvintsev et al. 
   
 The writing on this page is adapted from posts and discussion in the distill community slack group.  
 
+The [particle simulation](http://transdimensional.xyz/projects/neural_ca/index.html) code is [available as a google colab](https://colab.research.google.com/drive/1XaNCLrVyp5JYXgP_glWExSUyhbV8OaWb?usp=sharing)
```

Comment:
How about "Thanks to @anishau code to train the [particle simulation] model is available as a [colab notebook]"

Reply:
sounds good! I sent it back to ya for the minorest aesthetic thing - my social media brain is used to seeing '@' be a link to somwhere, but git markdown doesn't do that, wasn't sure what the standard was.